WordPress/gutenberg
# Fix VS Code performance

## What?

Improve VS Code performance by removing `jsconfig.json` file.

## Why?
If you open the project in VS Code and open a file like `test/native/integration/blocks-raw-handling.native.js`, you will see this warning:

<img width="510" alt="image" src="https://github.com/user-attachments/assets/21e1b470-76db-495e-8041-e9e57af4c33d" />


```log
To enable project-wide JavaScript/TypeScript language features, exclude large folders with source files that you do not work on.
```

The reason is the presence of `jsconfig.json` file that is outdated and hasn't been updated in 6 years apart from a formatting change 3 years ago.

VS Code seems to prefer `jsconfig` over `tsconfig` if it's present and thus that warning.

Now that the project has a good TS structure and a root `tsconfig`, we can get rid of that `jsconfig` to give VS Code some relief.


## How?

By nuking `jsconfig.json`

## Testing Instructions

- On trunk, open the project in VS Code
- Open a file like `test/native/integration/blocks-raw-handling.native.js`
- Confirm that you see that above mentioned warning.
- Now, on this branch, reload VS Code <kbd>Cmd/Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> > `Developer: Reload Window`
- Open the same file again
- Confirm that there is no warning

### Testing Instructions for Keyboard


## Screenshots or screencast 



|Before|After|
|-|-|
|||
